app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorUiRenderer.kt

```diff
@@ -1,9 +1,19 @@
 package org.simple.clinic.sync.indicator
 
 import org.simple.clinic.mobius.ViewRenderer
+import org.simple.clinic.sync.indicator.SyncIndicatorState.SyncPending
 
-class SyncIndicatorUiRenderer : ViewRenderer<SyncIndicatorModel> {
+class SyncIndicatorUiRenderer(val ui: SyncIndicatorUi) : ViewRenderer<SyncIndicatorModel> {
   override fun render(model: SyncIndicatorModel) {
-    //TODO: implement later
+    if (model.lastSyncedState == null)
+      return
```

Comment:
Can we either:

- Add braces here, or
- Move the `return` statement on the same line as the `if` clause

Reply:
This has been refactored and remove in later commits.